### DISCUSSION TITLE
Wordpress Installation

### DISCUSSION THREAD

**Discussion by daffanurrafiealam** [1Panel-dev/1Panel#7591] [General]
Hello, im newbie here.
![1panel](https://github.com/user-attachments/assets/1f31617b-5bd4-4233-9007-c83a5bf1b8d5)
How to add more than 1 websites using wordpress in 1panel?

**Comment by wanghe-fit2cloud** [comment]
Hello, WordPress is deployed using Docker containers, with each website running in its own dedicated WordPress container.

**Reply by daffanurrafiealam** [reply]
hello @wanghe-fit2cloud how to enable redis on wordpress with openresty? i tried installed php and install redis-server on app store but not detected on my wordpress site

**Reply by wanghe-fit2cloud** [reply]
When setting up a PHP runtime environment, you need to install Redis-related extensions, as shown in the image:
<img width="1475" alt="image" src="https://github.com/user-attachments/assets/667d5dc9-9765-41de-9572-741d8ff25fa9" />

Then, to access Redis in the website, use the container connection address:
<img width="1475" alt="image" src="https://github.com/user-attachments/assets/7cba95b1-4d5a-4908-a13f-4089264a90a6" />

**Reply by daffanurrafiealam** [reply]
hi @wanghe-fit2cloud PHP runtime environment already installed, but i still confused on second step, installed redis plugin on the wordpress but not detected, did i miss something? 
![redis](https://github.com/user-attachments/assets/a65cbd12-fff9-40ca-9095-858d5b2a06c1)

**Reply by wanghe-fit2cloud** [reply]
Both WordPress and Redis are deployed using Docker containers, so 127.0.0.1 cannot be used for cross-container access. 
The correct method to obtain the connection address is shown in the image below:
<img width="1475" alt="image" src="https://github.com/user-attachments/assets/7cba95b1-4d5a-4908-a13f-4089264a90a6" />

**Reply by daffanurrafiealam** [reply]
so i should bind the container connection?

**Reply by wanghe-fit2cloud** [reply]
Yes, you can give it a try.

**Comment by wan92hen** [comment]
You can use another port like 8081 to create a new WordPress website.

**Reply by daffanurrafiealam** [reply]
thank you, its work
